<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>端口：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_Port">
       <item>
        <property name="text">
         <string>USBAuto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ComAuto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB001</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>波特率：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_Rate">
       <property name="text">
        <string>115200</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_initPrint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>初始化</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>二维码信息：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_QRcode">
       <property name="text">
        <string>1123469-033</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>距离左边距(dot):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_LeftOffset">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>80</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>距离上边距(dot):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_TopOffset">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>打印信息</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btn_devStatus">
       <property name="text">
        <string>设备状态</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_PrintStatus">
       <property name="text">
        <string>打印状态</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_print">
       <property name="text">
        <string>打印</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>刷新状态</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_PagePrint">
       <property name="text">
        <string>页打印</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clearLog">
       <property name="text">
        <string>清空日志</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>状态信息：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_StatusStr"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>错误描述：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_StatusInfo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>日志</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QTextEdit" name="textEdit_log"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
